packages/components/pagination/src/pagination.ts

```diff
@@ -235,6 +237,22 @@ export default defineComponent({
       isAbsent(props.defaultCurrentPage) ? 1 : props.defaultCurrentPage
     )
 
+    const debounceMicroTask = (fn: () => void) => {
+      let scheduled = false
+      return () => {
+        if (scheduled) return
+        scheduled = true
+        Promise.resolve().then(() => {
+          scheduled = false
+          fn()
+        })
+      }
+    }
+
+    const handlePageChange = debounceMicroTask(() => {
+      emit('change', currentPageBridge.value, pageSizeBridge.value)
+    })
+
```

Comment:
Why need to use debounce?

Reply:
You want to delay triggering the change event after the nextTick.

Reply:
[10916](https://github.com/element-plus/element-plus/issues/10916)，I want to trigger the change event after nextTick to filter out redundant inaccurate sizeChange event.

Reply:
Yes emm, it seems not so elegant. It can be a workaround for the pagination which needs to be improved.